src/components/Header.tsx

```diff
@@ -0,0 +1,60 @@
+import styles from '../styles/Header.module.css';
+import CEOSLogo from '../../public/img/ceos-logo.svg';
+import Link from 'next/link';
+import { useRecoilState } from 'recoil';
+import { userState } from '@/store/store';
+import { useMutation } from '@tanstack/react-query';
+import { logout } from '@/apis/auth';
+import { useRouter } from 'next/router';
+
+export default function Header() {
+  const [user, setUser] = useRecoilState(userState); //전역 상태 userState
+  const { team: userTeam, part: userPart, name: userName, accessToken } = user; //구조분해할당
+
+  const router = useRouter();
+
+  //api 로직 가져와서 사용하기
+  const logoutMutation = useMutation(logout, {
+    onSuccess: data => {
+      console.log(data);
+      setUser({ ...user, accessToken: '', name: '', team: '', part: '' });
```

Comment:
`useResetRecoilState` method를 사용하면
```suggestion
  useResetRecoilState(userState);
```
로도 쓸 수 있어요!
userState의 초기값을 기억해두고 일일히 초기화하지않아도 되는 방법이라 추천드립니다~!